src/core/lib/debug/trace.cc

```diff
@@ -55,7 +55,7 @@ bool TraceFlagList::Set(const char* name, bool enabled) {
         found = true;
       }
     }
-    if (!found) {
+    if (!found && strcmp(name, "")) {
```

Comment:
The strcmp uses in this function could be decomposed into a separate function with clear naming so that the code is easier to read

Reply:
Can you elaborate here? c std lib calls are quite common, and pretty readable IMO.

Are you suggesting something like:

```
bool is_empty_string(const char* s) { return  !strcmp(s, ""); }
```

I am not inclined to wrap c lib calls unless there is extra functionality needed

Reply:
Also, I found the style guideline where I was taught to not wrap library calls without changing functionality: http://umich.edu/~eecs381/handouts/C_Coding_Standards.pdf

This is from a C class I took in college :)